ops/error_reporter.go

```diff
@@ -34,3 +36,38 @@ func PanicHandler(r ErrorReporter, next http.Handler) http.Handler {
 		next.ServeHTTP(w, req)
 	})
 }
+
+// Not sure why logrus doesn't just provide this...
+type logrusHook struct {
```

Comment:
If you want to keep this I suspect you'll prefer it in a different file, I can move it and add tests.

Reply:
how well does this fit with the assortment of glue code that you're collecting?

i see theoretical value in relaying log errors to the configured reporter, but so far keratin/authn-server has preferred sending errors directly to the reporter without going through logs (unless the reporter is LogReporter).

Reply:
This doesn't necessarily need to be a part of authn - I could implement it downstream.

I think reporting errors to both the logs and the error reporter makes most sense - yet it is clean to just pipe into the error reporter as you currently do. So I like automatically sending it to logger.

The case where it makes less sense of course is if the ErrorReporter is a LogReporter.